sessions/js-unit-testing/get-animal/index.test.js

```diff
@@ -0,0 +1,14 @@
+import {getAnimal} from './index.js';
+
+test('getAnimals() returns a string "I really like cats!" if called with the argument "cat".', () => {
+	const result = getAnimal('cat');
+	expect(result).toBe('I really like cats!');
+});
+test('getAnimals() returns a string "I like [animalPluralName]!" if called with a string argument.', () => {
+	const result = getAnimal('dog');
+	expect(result).toBe('I like dogs!');
```

Comment:
While working on this exercise my mind was boggling about building the plural form of an animal name. It's easy with "dog", but simple adding an "s" at the end wouldn't work with "goose". This might be an additional hurdle for students. 

I think we could overcome this issue by passing a plural form into the function (`getAnimal("dogs")`); Student would only need to write the logic to handle "undefined" and "cats" different from other inputs.

Reply:
I totally see your point here.

If we only use plural forms, there is one disadvantage: 
Inside the `getAnimal` function, students only have to check the prop to equal 'cats' or 'dogs', but they don't need to use the prop in the return statement as in `return <p>I totally like {animal}s.</p>`

If we can live with that and want the students to focus on understanding unit tests and write a simpler function, I am fine with that approach.